packages/checkout/widgets-lib/src/components/Handover/HandoverAnimation.tsx

```diff
@@ -0,0 +1,26 @@
+import { Box } from '@biom3/react';
+import React from 'react';
+import { Fit, Layout, useRive } from '@rive-app/react-canvas-lite';
+
+export function HandoverAnimation({ url }: { url: string }) {
+  const riveParams = {
+    src: url,
+    autoplay: true,
+    layout: new Layout({ fit: Fit.Contain }),
+    stateMachines: 'State',
+  };
+  const { RiveComponent } = useRive(riveParams);
+
+  // const riveAnimationState = useStateMachineInput(rive, 'State', 'mode', 0);
+  // console.log('Handover setup animation', id, riveAnimationState);
```

Comment:
Minor, comments

Reply:
Will clean this up in the next iteration to add in some animations.